Dockerfile

```diff
@@ -15,9 +15,10 @@ WORKDIR /app
 RUN \
     apt-get update \
     && apt-get dist-upgrade -y --no-install-recommends \
-    && apt-get install -y --no-install-recommends git openssh-client \
+    && apt-get install -y --no-install-recommends git openssh-client ca-certificates curl gnupg  \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/*\
+    && curl -fsSL https://get.docker.com | sh
```

Comment:
I am not fond of blindly pulling from get.docker.com. Since we only use `docker pull` and `docker save`, would it be enough to get the distro version instead?